newWebsiteDev/.browserslistrc

```diff
@@ -0,0 +1,13 @@
+# https://github.com/browserslist/browserslist#readme
```

Comment:
You can move all the files under a folder called bigpanther.ca instead of newWebsitedev